### DISCUSSION TITLE
Is there any information about China CI CD Workflow Guide?

### DISCUSSION THREAD

**Discussion by gaozuo** [codemagic-ci-cd/codemagic-docs#2877] [Q&A]
### Use case

```Text
I can't find any application ci cd workflow guide in China in the documentation or Google search. Can you provide some links to get started?
```

### Idea

```Text
Hope to add some documents and workflow guides and documents about the China region
```

**Comment by dam-ease** [comment]
Hello @gaozuo,
Currently, there isn’t a Codemagic CI/CD workflow guide specifically tailored for users in China or in chinese language. However, there is a comprehensive [documentation](https://docs.codemagic.io/) and guides to help set up CI/CD workflows for mobile app development, covering Android, iOS, and other platforms.
